### DISCUSSION TITLE
[Feature] Face recognition include age to determine faces

### DISCUSSION THREAD

**Discussion by jjspierx** [immich-app/immich#5690] [Feature Request]
### The feature

I would like Immich to use estimated age to help determine facial recognition.  Currently, Immich confuses my two daughters, and even my sons with me, from 40+ years ago.  It seems the facial recognition could be significantly more accurate if it used age and date of image as factors.

1.  Immich allows for entering date of birth for a person already.  This could be used to help determine people.  However, as far as I know there is no way to have to Immich re-run facial recognition without wiping out all previous data.  It might be nice to allow immich to re-run without wiping data, but to look at DOB's added for people to help increase the accuracy and make changes to detected faces/people.

2. Even withpout knowing DOB, it should be possible for Immich to look at a picture of a young child (lets say 5 years old) with an date taken in 2015, and assume that a photo with a date take on 1985 is not the same person.  And it currently does not do this.  I have children Immich confuses, that I think it would do a better job of tagging if it took the date of the picture into  account and could tell that a photos of a 5 year old should look drastically different than a photo of a 2 year old taken in the same year.

My feature request is that somehow the estimated age of the person in a photo as well as the date the photo was taken  could be passed into the ML process to help determine the person in the photo.

### Platform

- [x] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #4743
